network/dial/dial_task.go

```diff
@@ -0,0 +1,18 @@
+package dial
+
+import "github.com/libp2p/go-libp2p-core/peer"
+
+type DialTask struct {
+	index int
+
+	// info of the task
+	addr *peer.AddrInfo
```

Comment:
nitpick: var is called addr but it is addrInfo, I would call it addInfo

Reply:
[adb2138](https://github.com/0xPolygon/polygon-edge/pull/343/commits/adb2138d4318556d22e0900d20e4c3793949f3be)